### ISSUE TITLE
DAO Forum Governance

### ISSUE THREAD

**Issue by auryn-macmillan** [auryn-macmillan/DAOForum#10]
**How should the forum platform its self be governed? Who decides which features are implemented? Who decides which DAOs will be included?**

**Comment by ghost** [comment]
The big question. It should probably be DAO based. That being said it, it doesn't need to have thousands of members and its own token a la slock.it dao. For example, the MakerDAO guys (another interesting project) have a small, decentralized team that currently makes up their DAO. They have a functional set up whereby certain people are responsible for certain aspects that cater to their strengths.They have weekly teamspeak meetings that are open to the public and I've lurked, and I really dig the way they've managed themselves so far. 

Whether you want to call it a DAO or a team, I would vote for that to be the best way. Like I mentioned in the slock.it slack, we could even get funding from the slock.it DAO as a service provider with a well thought out proposal and an agreement to give them/us (I'm assuming we'll all be investing in the slock.it DAO anyway) a fixed % of all revenue + free use of the service in perpetuity. Then once it's up and running we could turn around and start offering the service to the various other DAOs that will be popping up over the next 12 months. Slack's pricing model + valuation is a good benchmark for the potential market ( https://slockit.slack.com/pricing + http://recode.net/2016/03/03/yes-slack-is-raising-money-but-will-it-get-the-5-billion-valuation-it-seeks-in-cloudy-funding-environment/). 

So yeah I'd gauge the interest and see how many people want to be a part. Seems like there's a few already and it's been <24h. I know I'm one :p

**Comment by CaptainSkywave** [comment]
I see a danger in slack - or other similar type - not being private (an employer can peek at your work slack eg.) and govs probably being able to monitor as well.
A true anonymous forum would be preferred.
I might be wrong here?

**Comment by auryn-macmillan** [comment]
Ideally I see this project as one which would be funded by the slock.it DAO solely for the DAO's benefit. Making it a generic service which is applicable to other DAOs would be in the slock.it DAO's best interest as it could provide an additional revenue stream.
Were it to be built in this way, I would think that the DAO should be the ultimate authority for this project. Having said that, the DAO would need to select a particular group of people (service provider) to run the project, and that service provider would need to have in house autonomy for day to day operations.

**Comment by vmichalik** [comment]
The issue I see is a closed-source project vs an open-source one. Isn't it in the best interest of DAO creators to have this kind of forum structure attached? If it's a core component won't someone else just make their own version? Getting enough contributors might be tricky.

**Comment by auryn-macmillan** [comment]
I think that's where price point needs to be considered. Basically you need to set the cost at a point where it makes more sense to another DAO to use your service rather than implement their own.

**Comment by vmichalik** [comment]
Ok. That makes sense- the service includes hosting and addressing new technical challenges etc. I think on-boarding for new DAOs is simplified if there's some mechanism to accept them. If we're a service provider, we'd be able to make that decision independently of the Slock.it DAO right? 

I think there's some blurring of how much this forum idea is aligned to the originating DAO and how much it's independent. How much can it really be distributed and autonomous if it's also a service provider?

**Comment by ghost** [comment]
I should clarify - in my first post I'm not advocating a closed-source solution. I think open source is the way to go, but even then there will need to be some form of governance and a team that guides the process. 

My main concern with having the slock.it DAO as the ultimate authority is that it could scare other DAOs away. Once the ecosystem is a little more mature and various DAOs have multiple projects, we will be in competition. Having your entire communication platform hosted on a service directly controlled by a competitor is not ideal. I think a good balance would be to get funding from the slock.it DAO, provide them with preferential rates and services for life and % of the revenue. The SP that gets created to run this project could be a separate open-source organization that serves the DAO community. That way we leverage having a close relationship with the slock.it DAO while also being a viable solution as an SP for other DAOs.

**Comment by auryn-macmillan** [comment]
@cjharty you're right, it would be beneficial for the project to remain neutral in given the potential for competition with other DAOs. So setting it up as a completely separate organisation and giving a percentage of revenue to the DAO who funds the project would align incentives well.
